encord/orm/label_row.py

```diff
@@ -260,6 +260,7 @@ class LabelRowMetadata(Formatter):
     data_title: str
     data_type: str
     data_link: Optional[str]
+    backing_item_uuid: str
```

Comment:
Known to be stricter and be a UUID however the other attributes on this object are all stringly typed so was following consensus.